### PR TITLE
writer title

### PR DESCRIPTION
**Pull request by valerii-fed**
body

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,21 +3,21 @@ test1
 test2
 test3
 test4
-test5
+
```
Comment (valerii-fed) [inline README.md]: Created by: **V.F.**
Time: **Sat Sep 22 12:37:38 EEST 2018**
Created by: V.F.
Time: Fri Sep 21 20:20:50 EEST 2018
Content: it's a new comment (val-fed)

Comment (valerii-fed) [inline README.md]: Created by: **V.F.**
Time: **Sat Sep 22 12:37:38 EEST 2018**
Created by: V.F.
Time: Fri Sep 21 20:20:53 EEST 2018
Content: it's a new comment (val-fed)

File: README.md [hunk 2]
```diff
@@ -3,21 +3,21 @@ test1
...
 test6
 test7
 test8
 test9
 test10
-test11
+edit-test11
 test12
 test13
 test14
```
Comment (valerii-fed) [inline README.md]: Created by: **V.F.**
Time: **Sat Sep 22 12:37:39 EEST 2018**
Created by: V.F.
Time: Fri Sep 21 20:20:56 EEST 2018
Content: iyujthygtfd

File: README.md [hunk 3]
```diff
@@ -3,21 +3,21 @@ test1
 test2
 test3
 test4
-test5
+
 test6
```
Comment (valerii-fed) [inline README.md]: Created by: **V.F.**
Time: **Sat Sep 22 12:37:39 EEST 2018**
Created by: V.F.
Time: Fri Sep 21 20:21:07 EEST 2018
Content: ....

Comment (valerii-fed) [inline README.md]: Created by: **V.F.**
Time: **Sat Sep 22 12:37:40 EEST 2018**
Created by: V.F.
Time: Fri Sep 21 20:21:32 EEST 2018
Content: val-fed comment

Comment (valerii-fed) [inline README.md]: Created by: **V.F.**
Time: **Sat Sep 22 12:49:16 EEST 2018**
```
code
code
code
code
code
```

File: README.md [hunk 4]
```diff
@@ -3,21 +3,21 @@ test1
 test2
 test3
```
Comment (valerii-fed) [inline README.md]: Created by: **V.F.**
Time: **Sat Sep 22 12:37:40 EEST 2018**
Created by: V.F.
Time: Fri Sep 21 20:21:37 EEST 2018
Content: a new comment (val-fed)

File: test.txt [hunk 5]
```diff
@@ -1,20 +1,17 @@
...
+
 test13
-test14
 test15
 test16
-test17
+edit-test17
 test18
 test19
 test20
```
Comment (valerii-fed) [inline test.txt]: Created by: **V.F.**
Time: **Sat Sep 22 12:37:41 EEST 2018**
Created by: V.F.
Time: Fri Sep 21 20:21:38 EEST 2018
Content: iuuykuykk

File: test.txt [hunk 6]
```diff
@@ -1,20 +1,17 @@
-test1
-test2
 test3
 test4
-test5
+edit
 test6
-test7
 test8
```
Comment (valerii-fed) [inline test.txt]: Created by: **V.F.**
Time: **Sat Sep 22 12:37:41 EEST 2018**
Created by: V.F.
Time: Fri Sep 21 20:21:40 EEST 2018
Content: edited comment

File: README.md (hunk 3 above)
Comment (val-fed) [reply README.md]: Reply `val-fed`

Comment (val-fed) [inline README.md]: Single comment `val-fed`

Comment (val-fed) [inline README.md]: review comment `val-fed`

**Pull request closed without merging**